doc/news/_preparation_next_release.md

```diff
@@ -275,7 +275,7 @@ This section keeps you up-to-date with the multi-language support provided by El
 - <<TODO>>
 - <<TODO>>
 - <<TODO>>
-- <<TODO>>
+- Denoted pacakage names & global variable names in [INSTALL.d](../INSTALL.md) as `Code` _(@janldeboer)_
```

Comment:
Typo here, it should be `[INSTALL.md]`

Reply:
Thank you for the note. I fixed the typo.